tests/accelerated_tests/CO_oxidation_Ruo2__acc_backend__do_acc_steps.py

```diff
@@ -432,15 +433,10 @@
                 tof_count={'CO_oxidation':-1})
 
 
-pt.export_xml_file(name+'.xml')
-pt.print_statistics()
-
-model_name = name
-import kmcos
+# Save the model to an xml file
 ###It's good to simply copy and paste the below lines between model creation files.
-pt.filename = model_name + ".xml"
```

Comment:
@meelod it is not surprising that this test would break.  You have simply pasted over the code here, when the acceleration option is needed. The '-t' option is needed within the compile_options, and you have not (as far as I know) added the compile_options code.  So this example needs to continue using the "export" command.